src/openms/include/OpenMS/FORMAT/DATAACCESS/SwathFileConsumer.h

```diff
@@ -542,6 +542,9 @@ namespace OpenMS
    * map) objects of MSDataCachedConsumer which can consume the spectra and
    * write them to disk immediately.
    *
+   * Warning: only the MS1 swathmap will be available then calling retrieveSwathMaps()
+   *          for downstream use.
```

Comment:
Hmmm I just checked this interface again and its a bit different than the rest as it will simply write out the spectra to mzML files and be done with it. Currently `retrieveSwathMaps` is not going to do what you would expect it to do and maybe the correct interface would be to have additional keep in memory parameters to specify whether MS1 or MS2 or both should be kept in memory.